### PR TITLE
feat: added AboutScreen

### PR DESCRIPTION
**Pull request by Fyuryy**
# What I did
Created About Screen

# How I did it
Created a new screen and tested it

# How to verify it
Go to settings screen and click on "About"

# Picture
Screenshot 2024-05-20 at 10 52 54

# Pre-merge checklist
The changes I have introduced:
- [x] work correctly
- [x] do not break other functionalities
- [x] work correctly on Android
- [x] are fully tested

### CODE REVIEW

**Comment by GustaveCharles** [comment]
Can you put at least a screenshot of what you added, so we can compare pleaase?

File: screens/Settings/AboutScreen/AboutScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+    "Gäel Conde",
+    "Alexandre Mourot",
+    "Gustave Charles",
+    "Gaspard Thoral",
+    "Pedro Laginhas",
+  ]
+
+  return (
+    <View style={[styles.container, { paddingTop: insets.top }]}>
+      <Text style={globalStyles.boldText}>About us</Text>
```
Comment (GustaveCharles) [inline screens/Settings/AboutScreen/AboutScreen.tsx:22]: The about us text layout is not good enough. There should be more free space around and it should be bigger

Comment (Fyuryy) [reply screens/Settings/AboutScreen/AboutScreen.tsx:22]: How can the text be bigger and still have more free space around it?

**Review by GustaveCharles — APPROVED** [review]
LGTM

**Pull request merged**